charts/codesealer/templates/core-manager-deployment.yaml

```diff
@@ -0,0 +1,62 @@
+{{- if .Values.manager.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+   {{- include "codesealer.labels" . | nindent 4 }}
+  name: {{ .Values.manager.name }}
+  namespace: {{ .Values.ingress.namespace }}
+spec:
+  replicas: {{ .Values.manager.replicaCount }}
```

Comment:
No reason to have exposed replica config for the manager, as it is inherently non-HA

Reply:
Will hardcode to 1